docs/guides/tournaments/hosting.md

```diff
@@ -26,39 +28,37 @@ If you want to run an event with more than 32 battlesnakes, you can simply creat
 <details><summary>How long do tournaments take to run?</summary>
 <p>
 
-#### That depends on the number of Battlesnakes in your tournament and how much time you give developers to build their Battlesnakes. That said, a typical tournaments with 16 Battlesnakes takes about 60-90 minutes to run. 
```

Comment:
These were just auto-corrected away 🤷 

I can add back in the periods if anyone has strong feels